### PR TITLE
Feature/admin election admin settings now save on change

### PR DESCRIPTION
**Pull request by ryyann**
To do:
- implement options settings for dates
- clean up UI

### CODE REVIEW

**Comment by ryyann** [comment]
tests pass on my machine, I'll investigate why circle rejects.

File: client/scripts/actions/ElectionActions.js [hunk 1]
```diff
@@ -3,46 +3,109 @@
 var Dispatcher = require('../dispatchers/default');
 var Constants = require('../constants/Constants');
 var EventEmitter = require('events').EventEmitter;
+var moment = require('moment');
 var API = require('../api');
 
+var ElectionStore = require('../stores/ElectionStore');
+var electionObj = require('../stores/emptyElectionObject');
+console.log(electionObj);
```
Comment (brianpchsu) [inline client/scripts/actions/ElectionActions.js]: Comment this out maybe?

File: client/scripts/actions/ElectionActions.js [hunk 2]
```diff
@@ -3,46 +3,109 @@
...
+    API.createElection(options);
+  },
+
+  // fetches all elections belonging to the current user
+  getUserElections: function() {
+    API.getUserElections();
+  },
+
+  setCurrentElection: function(id) {
+    API.getElectionById(id, true);
```
Comment (brianpchsu) [inline client/scripts/actions/ElectionActions.js]: It should be setElectionById right?

Comment (ryyann) [inline client/scripts/actions/ElectionActions.js]: nope. uses the method differently than just a get.

**Pull request merged**